### DISCUSSION TITLE
Update December 2022

### DISCUSSION THREAD

**Discussion by RevealedSoulEven** [rodit/SnapMod#293] [General]
https://github.com/RevealedSoulEven/SnapMod-new

Just a side project with some issues fixed and all features working fine😚
Please have a look

**Comment by jib0o** [comment]
![Screenshot_20221215-014604_Snapchat](https://user-images.githubusercontent.com/30224188/207731746-0b788b88-2b1d-42a3-9837-bbf82ef11343.png)

Thanks for what you do brother.
However, when a video of more than 60 seconds is shared, it is rejected.
It used to work

**Reply by RevealedSoulEven** [reply]
Didn't encountered this error bro

**Reply by Lucky-069** [reply]
Also I can read messages from notification even when message preview is turned off for notifications

**Reply by melbarouzi** [reply]
Facing Same bugs here

**Comment by RevealedSoulEven** [comment]
Lol 😂 I can't read even if it's on

**Comment by RevealedSoulEven** [comment]
I can't provide everything stable, but as far I know, the
notification/profile info are the only things causing bugs. No more

**Reply by melbarouzi** [reply]
You did a great job for us bro
Hope that  u keep updating this mod untill @rodit  comes back with new and enhanced  features

**Reply by RevealedSoulEven** [reply]
😿😿yeaa I'm too waiting for him. But I need help of someone who is experienced in modding, it's tough for me to find each and every function again if the pattern changes in updates😿
